data_handler/handlers/helpers.py

```diff
@@ -228,7 +232,14 @@ def load_data(
 
 
 # TODO: Improve this.
-def get_symbol(address: str) -> str:
+def get_symbol(address: str, protocol: str | None = None) -> str:
+    """
+    Get the symbol of the token by its address.
```

Comment:
update docstring, with adding info about sending error message to telegram bot

Reply:
Done